app-frontend/src/app/components/navBar/navBar.html

```diff
@@ -0,0 +1,48 @@
+<div class="navbar primary-navbar">
+	<!-- Nav Left -->
+	<div class="navbar-left">
+	  <a href="#" class="brand">
+			<img ng-attr-src="{{$ctrl.assetLogo}}">
```

Comment:
There are a lot of tabs spread throughout the files (in both html and scss). These should all be replaced with spaces before merging.